### PR TITLE
Adds /seed command (FS-225)

### PR DESCRIPTION
**Pull request by VideoGameSmash12**
This tries to recreate the vanilla command as closely as possible. Hopefully it does the trick.

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            TranslatableComponent response = new TranslatableComponent("commands.seed.success", seedString);
+            playerSender.spigot().sendMessage(response);
+        }
+        return true;
+    }
+
+    @Override
+    public List<String> getTabCompleteOptions(CommandSender sender, Command command, String alias, String[] args)
+    {
+        if (args.length <= 1)
```
Comment (Focusvity) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java:80]: This isn't really needed

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                world = playerSender.getWorld();
+            }
+            else
+            {
+                world = server.getWorlds().get(0);
+            }
+        }
+
+        // If the sender is not a Player, use the usual msg method to
+        if (isConsole())
```
Comment (Focusvity) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java:47]: The `senderIsConsole` variable is available to be used.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            if (world == null)
+            {
+                msg("That world could not be found", ChatColor.RED);
+                return true;
+            }
+        }
+        else
+        {
+            // If the sender is a Player, use that world. Otherwise, use the overworld as a fallback.
+            if (sender instanceof Player)
```
Comment (Focusvity) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java:36]: The `senderIsConsole` variable is available to be used.

**Review by Focusvity — CHANGES_REQUESTED** [review]

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java (hunk 1 above)
Comment (VideoGameSmash12) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java:80]: This was deliberate, it stops tab completion after an argument has been passed.

Comment (Focusvity) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java:80]: In that case, `==` is better used.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java (hunk 3 above)
Comment (VideoGameSmash12) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java]: Fixed

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java (hunk 2 above)
Comment (VideoGameSmash12) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java]: Fixed

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java (hunk 1 above)
Comment (VideoGameSmash12) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java]: Fixed

**Review by Focusvity — APPROVED** [review]

**Review by Focusvity — APPROVED** [review]
Please fix

**Pull request merged**